### PR TITLE
NightWatch Plugin feature

### PR DESCRIPTION
**Pull request by Abi1302**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,14 +136,14 @@ Note: The ChromeDriver version must match the Chrome browser version on your mac
```
Comment (princebaretto99) [inline README.md:136]: Update to browserstack.js and browserstack_local.js

File: package.json [hunk 2]
```diff
@@ -6,27 +6,24 @@
...
-		"docker-parallel": "nightwatch -c resources/conf/runners/docker.js --env chrome --parallel",
+		"bstack-parallel": "nightwatch -c resources/conf/runners/browserstack.js --env browserstack.chrome,browserstack.firefox,browserstack.edge --parallel",
 		"on-prem-suite": "nightwatch -c resources/conf/runners/on-prem.js",
-		"bstack-parallel-multiple": "nightwatch -c resources/conf/runners/browserstack.js --env chrome,edge,safari,android,iphone --parallel",
-		"bstack-local": "node ./resources/conf/runners/browserstack_local.js -c resources/conf/runners/browserstack.js --env chrome_local --tag single",
-		"bstack-local-parallel": "node ./resources/conf/runners/browserstack_local.js -c resources/conf/runners/browserstack.js --env chrome_local --parallel",
-		"bstack-local-parallel-multiple": "node ./resources/conf/runners/browserstack_local.js -c resources/conf/runners/browserstack.js --env chrome_local,edge_local,safari_local,android_local,iphone_local --parallel"
+		"bstack-parallel-multiple": "nightwatch -c resources/conf/runners/browserstack.js --env browserstack.chrome,browserstack.firefox,browserstack.edge --parallel",
```
Comment (princebaretto99) [inline package.json:15]: Update to use workers

**Review by princebaretto99 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Abi1302) [reply README.md:136]: done

File: package.json [hunk 3]
```diff
@@ -6,27 +6,24 @@
 	"scripts": {
 		"allure:generate": "allure generate -c results/**/",
 		"allure:open": "allure open",
-		"bstack": "nightwatch -c resources/conf/runners/browserstack.js --env chrome,edge,safari,android,iphone --parallel",
-		"docker": "nightwatch -c resources/conf/runners/docker.js --env chrome,firefox",
 		"on-prem": "nightwatch -c resources/conf/runners/on-prem.js",
 		"on-prem-parallel": "nightwatch -c resources/conf/runners/on-prem.js --parallel",
```
[the remark this answers is not in the document]
Comment (Abi1302) [reply package.json:10]: done

File: package.json (hunk 2 above)
Comment (Abi1302) [reply package.json]: Done

File: package.json [hunk 4]
```diff
@@ -6,27 +6,24 @@
...
 		"on-prem-suite": "nightwatch -c resources/conf/runners/on-prem.js",
-		"bstack-parallel-multiple": "nightwatch -c resources/conf/runners/browserstack.js --env chrome,edge,safari,android,iphone --parallel",
-		"bstack-local": "node ./resources/conf/runners/browserstack_local.js -c resources/conf/runners/browserstack.js --env chrome_local --tag single",
-		"bstack-local-parallel": "node ./resources/conf/runners/browserstack_local.js -c resources/conf/runners/browserstack.js --env chrome_local --parallel",
-		"bstack-local-parallel-multiple": "node ./resources/conf/runners/browserstack_local.js -c resources/conf/runners/browserstack.js --env chrome_local,edge_local,safari_local,android_local,iphone_local --parallel"
+		"bstack-parallel-multiple": "nightwatch -c resources/conf/runners/browserstack.js --env browserstack.chrome,browserstack.firefox,browserstack.edge --parallel",
+		"bstack-local": "npx nightwatch -c resources/conf/runners/browserstack_local.js --env browserstack.chrome --tag e2e",
+		"bstack-local-parallel": "nightwatch -c resources/conf/runners/browserstack_local.js -e browserstack.firefox,browserstack.chrome",
```
[the remark this answers is not in the document]
Comment (Abi1302) [reply package.json]: Done

**Review by princebaretto99 — APPROVED** [review]

**Pull request merged**